pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -182,11 +183,7 @@ func Pod(source *operatorsv1alpha1.CatalogSource, name string, image string, saN
 						},
 					},
 					SecurityContext: &corev1.SecurityContext{
-						ReadOnlyRootFilesystem:   pointer.Bool(false),
-						AllowPrivilegeEscalation: pointer.Bool(false),
-						Capabilities: &corev1.Capabilities{
-							Drop: []corev1.Capability{"ALL"},
-						},
```

Comment:
you can still drop the caps in all cases, right?  or did legacy catalog images require some additional caps?

Reply:
legacy catalogs did not require additional capabilities, but were the pods created by mentioning "drop all caps". So if our goal is to mimic the 4.10 configs, then what's there now is good. If that isn't our goal then I can test to make sure dropping all capabilities is fine before adding this back in.

Reply:
> legacy catalogs did not require additional capabilities, but were the pods created by mentioning "drop all caps". So if our goal is to mimic the 4.10 configs, then what's there now is good.

i'm confused.  in 4.10 did the catalog pods specify "drop all caps" or not?  We should align with what 4.10 did, imho.

Reply:
My apologies, I just realized I had a typo in my last comment, edited it so should make more sense now. 
Just to reiterate with more info: we DID NOT specify "drop all caps" for the pods: https://github.com/openshift/operator-framework-olm/blob/release-4.10/staging/operator-lifecycle-manager/pkg/controller/registry/reconciler/reconciler.go#L167-L169

Reply:
got it.